hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-nativetask/pom.xml

```diff
@@ -71,6 +71,11 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
why is this needed?

Reply:
One test in `hadoop-mapreduce-client-nativetask` uses `Lz4Codec`. If we don't add `lz4-java` as test dependency, because it is `provided` scope in `hadoop-common` now, we will get class not found exception.